alchemiscale/interface/api.py

```diff
@@ -102,6 +103,25 @@ def check_existence(
     return n4js.check_existence(scoped_key=sk)
 
 
+@router.post("/networks", response_model=ScopedKey)
+def create_network(
+    *,
+    network: Dict = Body(...),
+    scope: Scope,
+    n4js: Neo4jStore = Depends(get_n4js_depends),
+    token: TokenData = Depends(get_token_data_depends),
+):
+    validate_scopes(scope, token)
+
+    an = AlchemicalNetwork.from_dict(network)
+    an_sk = n4js.create_network(network=an, scope=scope)
+
+    # create taskhub for this network
+    n4js.create_taskhub(an_sk)
```

Comment:
No, `TaskHub`s are an internal detail to the state store; since an `AlchemicalNetwork` has one and only one of these, we use the `AlchemicalNetwork`'s `ScopedKey` to perform operations, such as actioning tasks, on its `TaskHub` via the `AlchemiscaleClient`.